plow_gui/README.md

```diff
@@ -0,0 +1,5 @@
+# Info
+
+The `plow_gui` crate is obsolete and usage of it is discouraged.
```

Comment:
Again, worthwhile a link to crate itself on crates.io.

Reply:
eedb96f